tests/common/fixtures/advanced_reboot.py

```diff
@@ -165,6 +173,8 @@ def __buildTestbedData(self, tbinfo):
         """
 
         self.mgFacts = self.duthost.get_extended_minigraph_facts(tbinfo)
+        if self.dual_tor_mode:
+            self.peer_mgFacts = self.peer_duthost.get_extended_minigraph_facts(tbinfo)
```

Comment:
What happens when `peer_duthost` is set to `None` in case of 
`self.peer_duthost = peer_duthost if peer_duthost != duthost else None`

Reply:
The dual tor warm reboot test will fail. Picking the peer ToR logic is covered in get_peerhost, removed if statement